packages/grid/src/vaadin-grid-mixin.js

```diff
@@ -270,6 +270,33 @@ export const GridMixin = (superClass) =>
         reorderElements: true,
       });
 
+      // Chromium based browsers cannot properly generate drag images for elements
+      // that have children with massive heights. This workaround prevents crashes
+      // and performance issues by excluding the items from the drag image.
+      // https://github.com/vaadin/web-components/issues/7985
+      document.addEventListener(
+        'dragstart',
+        (e) => {
+          // The dragged element can be the element itself or a parent of the element
+          if (e.target !== this && !e.target.contains(this)) {
```

Comment:
`!e.target.contains(this)` also covers `e.target !== this`

Reply:
Done.